### DISCUSSION TITLE
Requesting SC_VAT_TAX_REPORT always results in 403 Forbidden

### DISCUSSION THREAD

**Discussion by micamo2k17** [amzn/selling-partner-api-models#3633] [General]
Hi, I'm trying to migrate from MWS to SP-API.
With SP-API I'm able to request and download reports of other kinds (e.g.: ET_AMAZON_FULFILLED_SHIPMENTS_DATA_GENERAL), however I can't create reports of type SC_VAT_TAX_REPORT, since for every request I'm getting a 403 result with the "Access to the resource is forbidden" message. The "detail" field is empty so I have no indication of the possible cause.
I checked on SellerCentral, and all the roles are enabled for my application.
Any clue on what might be the reasons why this may happen?

Thank you.

**Comment by michelexport** [comment]
Hi,
you need to have restricted access role as developer.
Once you have been approved, you will be able to pull down datas every 12hrs.
